### ISSUE TITLE
Import context from golang.org/x/net

### ISSUE THREAD

**Issue by tbpg** [firebase/firebase-admin-go#110]
### [REQUIRED] Step 2: Describe your environment

  * Operating System version: N/A
  * Firebase SDK version: N/A
  * Library version: Master
  * Firebase Product: all

#### Steps to reproduce:

What happened? How can we make the problem occur?
This could be a description, log/console output, etc.

go version 1.6
`go get -u firebase.google.com/go`

Log from a failed Travis build:
```
get "firebase.google.com/go": found meta tag main.metaImport{Prefix:"firebase.google.com/go", VCS:"git", RepoRoot:"https://github.com/firebase/firebase-admin-go"} at https://firebase.google.com/go?go-get=1
firebase.google.com/go (download)
package context: unrecognized import path "context" (import path does not begin with hostname)
```
https://travis-ci.org/GoogleCloudPlatform/golang-samples/jobs/351066691#L1064

#### Relevant Code:

`import "context"` should be `import "golang.org/x/net/context"`.

This is mentioned in the [v.2.5.0 changelog](https://github.com/firebase/firebase-admin-go/blob/master/CHANGELOG.md#v250), but looks like there are some references remaining: https://github.com/firebase/firebase-admin-go/search?utf8=%E2%9C%93&q=context&type=. 

This is blocking https://github.com/GoogleCloudPlatform/golang-samples/issues/418.

I'll work on a PR.

**Comment by mbleigh** [comment]
I believe Go 1.6 is not supported by the Firebase Admin SDK. @hiranya911 to confirm.

**Comment by hiranya911** [comment]
Correct. We never really supported 1.6 properly, and starting from last release we officially dropped 1.6 support: #101 

Our documentation might be a little behind with regard to this change. I'll iron those our in the next few days.

**Comment by tbpg** [comment]
Any way to revisit that decision? I have [a PR](https://github.com/firebase/firebase-admin-go/compare/master...tbpg:context) ready which brings back compatibility.

Go 1.6 is still available in AppEngine, which we need to support in https://github.com/GoogleCloudPlatform/golang-samples/. In particular, I'm working on extending https://cloud.google.com/appengine/docs/standard/go/building-app/ to support authentication and it would be great to be able to use Firebase.

**Comment by hiranya911** [comment]
Why not use golang1.8 in App Engine? Given that Go no longer supports anything older than 1.9 (see https://golang.org/doc/devel/release.html), it seems unnecessary to support something as old as 1.6.

**Comment by tbpg** [comment]
AppEngine Standard still has 1.6 available.

**Comment by hiranya911** [comment]
That is true, but developers can choose 1.8 when using Firebase. Is there any reason that takes that choice out of the picture?

**Comment by tbpg** [comment]
Some users are stuck on older versions of Go without an easy ability to upgrade. So, for https://github.com/GoogleCloudPlatform/golang-samples/, we support all possible versions of Go.

**Comment by hiranya911** [comment]
In #98 we found that the Admin SDK never really supported Go 1.6. What's interesting is that we didn't hear about this from any users. We discovered it ourselves while setting up CI for Go 1.6. So I think it's fair to assume that all Admin SDK users are already on Go 1.7 or higher. Given that, and all the internal discussions we've had on this topic, I'm reluctant to go back. 

But if you're willing to provide a PR, with the necessary code changes and a working Travis config for 1.6, then we can revisit this.
